scripts/manifest/generate.R

```diff
@@ -0,0 +1,42 @@
+#' Write out templated Windows Application Manifest files
+#'
+#' We use two files to embed an application manifest on Windows:
+#' - `{crate}-manifest.rc`
+#' - `{crate}.exe.manifest`
+#'
+#' And we generate these two files for both `harp` and `ark`.
+#'
+#' They are effectively the same, we just need to swap out the crate names,
+#' so we use this script to write the files in a consistent manner.
+write_manifest <- function(root, crate) {
```

Comment:
Because the manifest files are almost identical between ark and harp, we generate them from templates